# Obstacle feed mode (0 = send detected obstacles through as ellipses. 1 = publish predefined ellipses)
# need to recompile after changing!
obstacle_feed_mode: 1

# Distance within which obstacles should be send through
distance_threshold: 10
obstacle_threshold: 10
max_obstacle_radius: 8

# Rate at which ellipse obstacles should be send out
update_rate: 5

# Define the frame in which the predefined obstacles are represented
#obstacle_base_link: base_link
obstacle_base_link: odom

pub_ellipse_topic: ellipse_objects_feed
pub_visualization_topic: visualize_ellipses_array 
